docs/how-to-develop-scraper.md

```diff
@@ -200,7 +200,7 @@ You can check whether your scraper is passing the tests by running
 $ python -m unittest -k <test_file_name>
 ```
 
-where `myscraper` is the name of the json file without the `.json` extension.
+where `test_file_name` is the name of the test data json file without the `.json` extension in `tests/test_data/` directory.
```

Comment:
I'd probably recommend an alternative route here; the command-line argument isn't really a filename, so how about:

```suggestion
$ python -m unittest -k <ClassName.lower()>

Where `ClassName` is the name that you used earlier to generate the scraper.
```

Reply:
Good suggestion. While my arephrasing was better than the original one it was still a bit too clumsy.